api/websocket/messagebuffer/messagebuffer_test.go

```diff
@@ -0,0 +1,129 @@
+package messagebuffer
+
+import (
+	"encoding/hex"
+	"testing"
+	"time"
+
+	"github.com/nknorg/nkn/v2/pb"
+	"github.com/stretchr/testify/require"
+)
+
+var clientId [][]byte = [][]byte{[]byte{byte(0x01)}, []byte{byte(0x02)}, []byte{byte(0x03)}, []byte{byte(0x04)}}
+
+var msg *pb.Relay = &pb.Relay{SrcIdentifier: "from", DestId: []byte("to"), Payload: []byte("payload"),
```

Comment:
This test data cannot cover expiration logic because they all have the same `MaxHoldingSeconds`

Reply:
Yes. When testing removeExpired(), it use time.Sleep to wait all message expired.

Reply:
I add an function to get random MaxHoldingSeconds.